app/src/app/[lng]/page.tsx

```diff
@@ -28,15 +28,131 @@ import { FiDownload } from "react-icons/fi";
 import {
   MdArrowForward,
   MdArrowOutward,
+  MdCheckCircleOutline,
   MdGroup,
   MdOutlineAddchart,
   MdOutlineAspectRatio,
 } from "react-icons/md";
 import { PiTrashLight } from "react-icons/pi";
 import { TbBuildingCommunity } from "react-icons/tb";
+import { useToast } from "@chakra-ui/react";
 
 export default function Home({ params: { lng } }: { params: { lng: string } }) {
   const router = useRouter();
+  const toast = useToast();
+  enum STATUS {
+    INFO = "info",
+    SUCCESS = "success",
+    ERROR = "error",
+  }
+
+  const CITY_INTENTORY_YEAR = "DE_BER";
+
+  const showToast = (
```

Comment:
Extract this outside of the function as well, so it's above Home. Could be extracted into another file later if necessary.